actions/plan/comment.sh

```diff
@@ -21,7 +21,7 @@ if [ "${HAS_CHANGES}" == "false" ]; then
 	exit 0
 fi
 
-PLAN_TEXT=$(terraform show "${ARTIFACTS_DIR}/terraform.plan" -no-color)
+PLAN_TEXT=$(terraform show "${ARTIFACTS_DIR}/terraform.plan" -no-color | sed --silent '/Terraform will perform the following actions/,$p')
```

Comment:
Note that there's also `refresh_on_pr: false` which you might be interested in setting, depending on your workload.

Reply:
(semi-related)